index.html

```diff
@@ -1,1227 +1,1579 @@
+						media="(min-width:480px)">
+					<img class="intro__image_bg" src="img/intro/background/x320/bgIntroFull_x320.jpg" alt="background">
+				</picture>
+				<picture>
+					<source srcset="img/intro/background/x1920/bushLeft_x1920.png" type="image/jpeg"
+						media="(min-width:1920px)">
+					<source srcset="img/intro/background/x1000/bushLeft_x1000.png" type="image/jpeg"
+						media="(min-width:1000px)">
+					<source srcset="img/intro/background/x768/bushLeft_x768.png" type="image/jpeg"
+						media="(min-width:768px)">
+					<source srcset="img/intro/background/x480/bushLeft_x480.png" type="image/jpeg"
+						media="(min-width:480px)">
+					<img class="intro__image_bushLeft" src="img/intro/background/x320/bushLeft_x320.png"
+						alt="cloudsBottom">
+				</picture>
+				<picture>
+					<source srcset="img/intro/background/x1920/bushRight_x1920.png" type="image/jpeg"
+						media="(min-width:1920px)">
+					<source srcset="img/intro/background/x1000/bushRight_x1000.png" type="image/jpeg"
+						media="(min-width:1000px)">
+					<source srcset="img/intro/background/x768/bushRight_x768.png" type="image/jpeg"
+						media="(min-width:768px)">
+					<source srcset="img/intro/background/x480/bushRight_x480.png" type="image/jpeg"
+						media="(min-width:480px)">
+					<img class="intro__image_bushRight" src="img/intro/background/x320/bushRight_x320.png"
+						alt="cloudsBottom">
+				</picture>
+			</div>
+			<div class="intro__container">
+				<header class="intro__header">
+					<div class="header__logo">
+						<a href="#">
+							<img class="header__logo_img" src="img/intro/introHeader/logo.svg" alt="logo">
+						</a>
+					</div>
+					<button class="menu_burger">
+						<div class="menu_burger_button">
+							<div class="burger_button_line"></div>
+							<div class="burger_button_line"></div>
+							<div class="burger_button_line"></div>
 						</div>
-					</header>
-					<div class="intro__content" id="intro__content">
-						<h1 class="intro__invitation">Окунитесь в мир лошадей вместе с нами</h1>
-						<div class="intro__advertising">
-							<p>Уроки верховой езды, фотосессии и мероприятия в частном конном клубе <b>г. Киев</b> </p>
+						<div class="menu_burger_name">
+							<p>Меню</p>
 						</div>
-						<button class="intro__button button popup-link" data-btn-content="записаться">
-							записаться
-						</button>
+					</button>
+					<nav class="header__nav">
+						<ul class="nav__menu_header">
+							<li><a class="menu__link_header" href="#">О нас</a></li>
+							<li><a class="menu__link_header" href="#">Услуги и цены</a></li>
+							<li><a class="menu__link_header" href="#">Фотоальбом</a></li>
+							<li><a class="menu__link_header" href="#">Контакты</a></li>
+						</ul>
+					</nav>
+					<div class="header__contacts" data-timetable="Ежедневно 9:00 – 22:00">
+						<a class="tel_number" href="tel:+38 (000) 000-00-00">
+							<img class="tel_number_img" src="img/intro/introHeader/tel.svg" alt="tel_icon">
```

Comment:
read what is alt attribute and check if you really need to join words in it

check all the places

Reply:
Done.

Reply:
Considering language of your site, I think there must be not English words

Reply:
Done.